### DISCUSSION TITLE
m2e Tycho Project Configurators for Tycho 2.5.0?

### DISCUSSION THREAD

**Discussion by sewe** [eclipse-tycho/tycho#324] [Q&A]
Hi all,

I am currently in the process of making the jump from Tycho 2.0.0 to 2.5.0.

Unfortunately, using the Tycho Project Configurators 0.8.1.201704211436
(`org.sonatype.tycho.m2e.feature.feature.group`) I now get errors like the
following:

> Plugin execution not covered by lifecycle configuration: org.eclipse.tycho:target-platform-configuration:2.5.0:target-platform (execution: default-
>  target-platform, phase: initialize)

Is the `target-platform` goal new since 2.0.0?

Of course, I could simply add an `<ignore/>` pseudo-`<configuration>` for
that particular goal, but as Tycho worked out of the box with m2e
before, this seems less than satisfactory.

But maybe I am doing something wrong. Any pointers as to what?

Best wishes,

Andreas

**Comment by laeubi** [comment]
See for example https://github.com/eclipse/tycho/issues/146

**Reply by sewe** [reply]
Not sure what you suggest I do to get rid of the warning? What for a future version of Tycho? Of m2e? Or something else?

**Reply by laeubi** [reply]
Currently there might be warnings for (new) tycho versions, and we are looking for providing better m2e support here.

**Reply by cdietrich** [reply]
in xtext we workarounded the problem by adding yet another ignore https://github.com/eclipse/xtext-core/pull/1725/files

**Comment by mickaelistria** [comment]
Can you please check what emits this warning? Sharing the "Type" of this marker from the Problems view would be perfect.

**Reply by guw** [reply]
The warning is from M2E.

Example:
```
Description	Resource	Path	Location	Type
Plugin execution not covered by lifecycle configuration: org.eclipse.tycho:tycho-compiler-plugin:2.3.0:compile (execution: default-compile, phase: compile)	pom.xml	/org.eclipse.jdt.ls.core	line 11	Maven Project Build Lifecycle Mapping Problem
```

There is a QuickFix to discover new M2E connectors but it's not finding anything for Tycho.
